libraries/AP_Logger/LogStructure.h

```diff
@@ -1264,13 +1264,9 @@ LOG_STRUCTURE_FROM_AVOIDANCE \
       "SIM","QccCfLLffff","TimeUS,Roll,Pitch,Yaw,Alt,Lat,Lng,Q1,Q2,Q3,Q4", "sddhmDU????", "FBBB0GG????" }, \
     { LOG_TERRAIN_MSG, sizeof(log_TERRAIN), \
       "TERR","QBLLHffHH","TimeUS,Status,Lat,Lng,Spacing,TerrH,CHeight,Pending,Loaded", "s-DU-mm--", "F-GG-00--" }, \
-    LOG_STRUCTURE_FROM_ESC_TELEM \
+LOG_STRUCTURE_FROM_ESC_TELEM \
```

Comment:
Whitespace....

And the two last commits should be squashed, looks like a rebase caused the previous commit to get messed you.

Reply:
This was intentional - all the other imported structures are indented like this